apps/ui/package.json

```diff
@@ -26,6 +26,9 @@
     "@fontsource/roboto": "5.0.8",
     "@graphql-debugger/time": "workspace:^",
     "@graphql-debugger/utils": "workspace:^",
+    "@tanstack/react-query": "^4.35.7",
+    "@tanstack/react-query-devtools": "^4.35.7",
+    "classnames": "^2.3.2",
```

Comment:
```suggestion
    "@tanstack/react-query": "4.35.7",
    "@tanstack/react-query-devtools": "4.35.7",
    "classnames": "2.3.2",
```

Let's pin these as we are shipping to the public and floating could cause more problems.

Reply:
FYI - is `@tanstack/react-query-devtools` for dev only?

Reply:
I made two commits to pin/lock all.